docs/cas-server-documentation/installation/Configuration-Properties.md

```diff
@@ -3446,6 +3446,8 @@ prefixes for the `keystorePath` or `identityProviderMetadataPath` property).
 # cas.authn.pac4j.saml[0].identityProviderMetadataPath=
 
 # (Optional) Friendly name for IdP, e.g. "This Organization" or "That Organization"
+# This name, with 'nonword' characters converted to '-' (e.g. This Org (New) becomes This-Org--New- ),
```

Comment:
Could you document the same sort of explanation in the code where the property exists? in form of javadocs?